conda_smithy/linter/lints.py

```diff
@@ -983,3 +987,100 @@ def sort_osx(versions):
         ):
             if sdk_lint not in lints:
                 lints.append(sdk_lint)
+
+
+def lint_recipe_is_parsable(
+    recipe_text: str,
+    lints: List[str],
+    hints: List[str],
+    recipe_version: int = 0,
+):
+    parse_vars = []
+    parser_names = []
```

Comment:
I think this should be a `dict` so we don't rely on the implicit pairing of the values.

Reply:
done!